frontend/src/ClientDataTable.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import './App.css'
+
+
+const ClientDataList = () => {
+    const [data, setData] = useState([]); //
+      
+        //Fetch data using axios
+    useEffect(() => {
```

Comment:
We might also want to consider using `useQuery` instead of `useEffect` to fetch data: https://johnnie-agonz71.medium.com/using-reacts-usequery-hook-to-fetch-data-cf25ee334f25